src/platform_impl/windows/mod.rs

```diff
@@ -13,13 +13,15 @@ pub use self::icon::WinIcon as PlatformIcon;
 
 use crate::event::DeviceId as RootDeviceId;
 use crate::icon::Icon;
+use crate::window::Theme;
 
 #[derive(Clone)]
 pub struct PlatformSpecificWindowBuilderAttributes {
     pub parent: Option<HWND>,
     pub taskbar_icon: Option<Icon>,
     pub no_redirection_bitmap: bool,
     pub drag_and_drop: bool,
+    pub forced_theme: Option<Theme>,
```

Comment:
I don't really like the word `force` here, maybe we should use `prefer` and in some other places?

```suggestion
    pub preferred_theme: Option<Theme>,
```